src/css/main.less

```diff
@@ -1478,3 +1472,7 @@ h3.small-caps {
     padding: 0;
   }
 }
+
+.task-details dt {
+  text-align: left;
```

Comment:
I thought about it but feared unwanted side-effects (and I was thinking about the About modal)...